<?xml version="1.0"?>

<launch>
  <!-- Configurable args -->
  <arg name="input_topic" default="/profiles" />
  <arg name="input_type" default="pointcloud2" />
  <arg name="target_frame" default="mur620c/base_footprint" />
  <arg name="source_frame" default="" />
  <arg name="output_topic" default="/assembled_pointcloud" />
  <arg name="max_points" default="5000000" />

  <node pkg="laser_scanner_tools" type="pointcloud_assembler.py" name="pointcloud_assembler" output="screen">
    <param name="input_topic" value="$(arg input_topic)" />
    <param name="input_type" value="$(arg input_type)" />
    <param name="target_frame" value="$(arg target_frame)" />
    <param name="source_frame" value="$(arg source_frame)" />
    <param name="output_topic" value="$(arg output_topic)" />
    <param name="max_points" value="$(arg max_points)" />
  </node>
</launch>
